src/controllers/authController.js

```diff
@@ -33,6 +63,33 @@ const register = async (req, res) => {
 };
 
 const login = async (req, res) => {
+  /* 
+    #swagger.summary = 'Login as a user'
+    #swagger.parameters['user'] = {
+      in: 'body',
+      description: 'User email and password for login',
+      required: true,
+      schema: {
+        type: 'object',
+        properties: {
+          email: { type: 'string' },
+          password: { type: 'string' }
+        }
+      }
```

Comment:
```suggestion
      schema: {
        email: "jdoe@example.com",
        password: "supersecretpassword"
      }
```

Reply:
<img width="330" alt="schema" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/e963cf6e-facc-4f6a-bb96-267a37e24f08">